A/Antlr4CppRuntime/build_tarballs.jl

```diff
@@ -41,4 +39,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"11")
```

Comment:
GCC 10 worked before, no?
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"10")
```

Reply:
apple-darwin seemed to fail with v10
https://buildkite.com/julialang/yggdrasil/builds/967#01865a14-2d2b-4a05-8696-75d6bbbbcd5f

Reply:
1. we build with Clang, not GCC, on BSD platforms (macOS and FreeBSD)
2. even if we used GCC, `aarch64-apple-darwin` has only one GCC toolchain, which is version 12, so that using asking for version 10 or 11 is totally irrelevant, and this is already made irrelevant by the previous point

Reply:
Thanks for all the help! Although I have no idea how to proceed.